CONTRIBUTING.md

```diff
@@ -0,0 +1,140 @@
+# Contribution Guidelines
+
+First of all, thank you for taking your time to contribute to `wasmi`!
+
+Reading these contribution guidelines is your best initial step towards
+sucessfully driving the development of `wasmi` forward with your own ideas
+and use cases.
+
+## Code of Conduct
+
+Please respect our [code of conduct](./CODE_OF_CONDUCT.md) in every
+communication and discussion related to the `wasmi` project.
+
+## I don't want to contribute, I just have some questions
+
+For technical questions about `wasmi` feel free to contact
+us via one of the following communication channels:
+
+- GitHub Discussions
+    - Write a [**new GitHub Discussions post**](https://github.com/paritytech/wasmi/discussions/new).
+    - For simple questions around usage or development of `wasmi`.
```

Comment:
Let's not do github discussion and disable them. They are hard to discover and will only be seen by people following the repo. Better make them go to the forum for discussion.

Reply:
I want to try out and evaluate GitHub Discussions for a while since it is important to open up `wasmi` for use cases outside of Polkadot smart contracts. If it turns out to be a mess we can still disable it.